conans/client/tools/net.py

```diff
@@ -3,13 +3,13 @@
 import os
 from conans.client.output import ConanOutput
 from conans.client.rest.uploader_downloader import Downloader
-from conans.client.tools.files import unzip, check_md5, check_sha1, check_sha256
+from conans.client.tools.files import unzip, check_md5, check_sha1, check_sha256, verify_gpg_sig
 from conans.errors import ConanException
 
 _global_requester = None
 
 
-def get(url, md5='', sha1='', sha256='', destination="."):
+def get(url, md5='', sha1='', sha256='', gpg_signature='', gpg_pubkey='', destination="."):
```

Comment:
I am a bit confused about the input parameters, and the underlying verify_gpg_sig() function. This assumes:
- always using "keys.gnupg.net" as server?
- If using a file for pubkey, what is the path of the file? Relative to the current folder? The file has been ``exports`` with the recipe?

Reply:
I mean, we could allow the user to supply their own keyserver for completenes, but the major ones are all synced. 

I would envisage the public key file is included in the recipe and exported.